src/main/java/frc/robot/commands/shooter/PivotMove.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.commands.shooter;
+
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.RobotConfig.ShooterConfig;
+import frc.robot.subsystems.shooter.Shooter;
+
+public class PivotMove extends Command {
```

Comment:
Should this command extend an execute() and/or end() method?

Reply:
no